python/dgl/utils/data.py

```diff
@@ -147,57 +151,62 @@ def graphdata2tensors(data, idtype=None, bipartite=False):
         num_src, num_dst = infer_from_raw
     return src, dst, num_src, num_dst
 
-def networkxbipartite2tensors(nx_graph, idtype, edge_id_attr_name='id'):
+def networkxbipartite2tensors(nx_graph, idtype, top_map, bottom_map, edge_id_attr_name=None):
     """Function to convert a networkx bipartite to edge tensors.
 
     Parameters
     ----------
     nx_graph : nx.Graph
         NetworkX graph. It must follow the bipartite graph convention of networkx.
         Each node has an attribute ``bipartite`` with values 0 and 1 indicating
-        which set it belongs to. Only edges from node set 0 to node set 1 are
-        added to the returned graph.
+        which set it belongs to.
+    top_map : dict
+        The dictionary mapping the original node labels to the node IDs for the source type.
+    bottom_map : dict
+        The dictionary mapping the original node labels to the node IDs for the destination type.
```

Comment:
I don't know whether this networkx's bipartite to DGL graph is used very frequently. I feel we don't need to add more functionality to it unless absolutely needed. Since you've added this, you can leave it there.